account_payment_term_cutoff_day/tests/test_account_invoice_date_due.py

```diff
@@ -45,6 +45,17 @@ def test_due_date_after_cutoff_day(self):
             expected_due_date,
         )
 
+    def test_due_date_after_cutoff_day_with_31_days(self):
+        """Test if due date is shifted by +1 month and the following month has 31 days
+        when invoice date is after cutoff_day"""
+        invoice_date = date(2025, 3, 21)
```

Comment:
Since the implementation is changed, I think we should test with a date like 31 Jan, instead.